components/inputs/test/input-number.test.js

```diff
@@ -399,7 +399,7 @@ describe('d2l-input-number', () => {
 		].forEach((test) => {
 			it(test.name, async() => {
 				const elem = await fixture(test.fixture);
-				if (test.value !== null) elem.value = test.value;
+				if (test.value !== null && test.value !== undefined) elem.value = test.value;
```

Comment:
When I was working on adding a test I noticed that if I had no value in my test object then it was setting the elem value to undefined (but I had actually defined the value on the fixture), though maybe it was done this way on purpose? Otherwise having the positive case (`if (test.value) elem.value = test.value`) might be more clear